redbot/core/bot.py

```diff
@@ -216,7 +216,6 @@ async def prefix_manager(bot, message) -> List[str]:
 
         self._uptime = None
         self._checked_time_accuracy = None
-        self._color = discord.Embed.Empty  # This is needed or color ends up 0x000000
```

Comment:
Should we be assigning this attribute outside of `__init__`?

Reply:
No 3rd-party code executes before it's set in _pre_login() so it's fine. Same as with self.description